src/WordPress/Hooks.php

```diff
@@ -18,8 +18,8 @@ class Hooks
 
     public function __construct()
     {
-        $this->config = new Integration\DefaultConfig('[]');
-        $this->logger = new Integration\DefaultLogger(false);
+        $this->config = new Integration\DefaultConfig(file_get_contents(CLOUDFLARE_PLUGIN_DIR.'config.js', true));
+        $this->logger = new Integration\DefaultLogger($this->config->getValue('debug'));
```

Comment:
Does this actually turn on debug logging?  I feel like when we switched to the PSR logger we removed this because you had to turn it on some other way.